src/utils/restApiWrapper.ts

```diff
@@ -0,0 +1,58 @@
+export async function postData(
+    url = '',
```

Comment:
Since most of the calls will be made to the rucio server, what do you think about replacing url with host and endpoint? i.e. the `scheme://hostname:port` for the rucio server could be picked up from the environment variables.
So, we could have two function arguments, `host` which contains the default scheme+host+port to use ( rucio server host). and `endpoint` which contains the relative path.

Reply:
Yes, made the necessary changes!